modules/3d/include/opencv2/3d/detail/kinfu_frame.hpp

```diff
@@ -0,0 +1,20 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html
+
+// This code is also subject to the license terms in the LICENSE_KinectFusion.md file found in this module's directory
```

Comment:
There is no "module directory" for installed binaries.

Reply:
Inserted license text instead of this notion as it's done with SoftFloat or FLANN.

Should I left it like that? Or, if not, where to put the license texts so that they'd deploy with installed files?

Reply:
* WillowGarage license notions removed since it's already in OpenCV's `COPYRIGHT` file
* KinFu license notion shortened